### ISSUE TITLE
The Markdown citation dropdown selector only appears for Markdown primary citation template e.g., inside square brackets

### ISSUE THREAD

**Issue by MSBack** [aidenlx/obsidian-zotero#114]
I would expect the markdown citation dropdown to be triggered by the `@`, but in reality, it requires `[@` which is a bit cumbersome to type for every citation.

See this demo:

![Obsidian_MZWUAZKhrv](https://user-images.githubusercontent.com/33608083/228462572-2b489d71-0b71-4669-99a5-b4fe4716855f.gif)

Another related issue is that shift+enter just goes to the next line, rather than insert the secondary markdown citation.

**Comment by MSBack** [comment]
Any update on this, @aidenlx ?

**Comment by FeralFlora** [comment]
With the recent updates, I think this and #117 are two of the main friction points remaining before Zotlit becomes the obvious choice for integrating Zotero with Obsidian. The field template, and bug resolutions, have made the literature note feature quite satisfactory, but I think due to the aforementioned issues, the citation workflow can still be improved.

**Comment by FeralFlora** [comment]
Bumpy

**Comment by FeralFlora** [comment]
Bump

**Comment by FeralFlora** [comment]
bump

**Comment by dverslegers** [comment]
Agreed, having this option and a bit more customisability for the citation templates would be fantastic!

**Comment by aidenlx** [comment]
will be addressed in v2 beta.2: https://github.com/aidenlx/zotlit/commit/64921c29d5eebc668c1b6fc2f4bb98a1d5528243

v2 beta is available now: https://zotlit.aidenlx.site/docs/install-zotlit
Migration guide: https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
